### DISCUSSION TITLE
Done Laying Out the Product Requirements

### DISCUSSION THREAD

**Discussion by corygalyna** [ZORALab/demo-web-prestige#47] [Q&A] [answered]
Don't see we need to go further than here. It's suffice. Anything else, @hollowaykeanho ?

**Comment by hollowaykeanho** [comment, marked as answer]
i18n?

**Comment by corygalyna** [comment]
Oh yeah... totally forgot. Will add it in.

**Comment by corygalyna** [comment]
Ok done. Please work them out. It's okay.. you can decide on behalf of me as long as we can win the contract.

**Comment by hollowaykeanho** [comment]
Got it.
